### ISSUE TITLE
Use 

### ISSUE THREAD

**Issue by goldibex** [johnpapa/lite-server#15]
Thanks so much for building this tool, it's really outstandingly useful.

In keeping with the "the future is today" mentality of most of the newer JS frameworks, would you be willing to entertain a PR that introduced support for HTTP/2 and SPDY via [node-spdy](https://github.com/indutny/node-spdy)?

**Comment by cgmartin** [comment]
As a simple development server I don't feel like this has much value. Is there a particular use-case for development that you have that requires HTTP/2? 

Running `lite-server` in production is absolutely not recommended. :)

Another quirk is that this project is a simple wrapper around [BrowserSync's](https://www.browsersync.io/) server feature. If it can't do HTTP/2, then I assume we'd have large difficulty trying to work that in via `lite-server`. But if Browsersync ever does include that feature, it could be easily enabled through lite-server's overrides.

**Comment by johnpapa** [comment]
I agree with Chris, but curiosity has got me ... I'd like to see what you have in mind just to satisfy that :)

**Comment by goldibex** [comment]
Thanks all. Well, let me lay out my arguments for you here first -- if
they're totally unconvincing it may not be worth the effort.

1. lite-server is part of the "future, now" Javascript ecosystem. It's
advertised for use developing with JSPM and Angular 2, for instance. HTTP/2
is the future of communication on the Web, so it makes sense to get people
developing on that now.

2. I have some very, very preliminary data showing that in development
cases where hundreds of objects need to be loaded from the server (common
enough with JSPM and even modestly sized projects these days), HTTP/2 can
cut down time to first load for developers considerably.

3. It's a little-ish issue, but I think it's also important to get
developers saying the mantra "SSL only," starting from their development
instances. I've already developed a tool that de-complicates the process so
you don't have to worry about bright red SSL warnings more than, say, once
a month. It's true that you could do the same thing against HTTP/1.1 also,
but packaging it with a move to HTTP/2 makes it easier for developers to
swallow -- honey on the rim of the medicine cup, as it were.

If this is at least a colorable argument to you, let me know and I'll take
a pass at it.

All best
Harry

**Comment by jakeNiemiec** [comment]
+1

**Comment by Meligy** [comment]
Another +1 for this, especially as a development server.

When you load Angular2 via SystemJS, you have quite a large number of requests depending on your import tree. 

This is not the production experience, but it kinda slows down the development experience ([sample](https://medium.com/@martin_hotell/angular-cli-and-real-development-productivity-4ba3c1865eda#.9igwsyrcl)).

HTTP/2 might be a good solution for this development-only problem.

**Comment by jperezruiz** [comment]
+1
